src/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Ekaterina developer</title>
+  <script src="./formatDate.js"></script>
+</head>
+<body>
+  
+</body>
+</html>
\ No newline at end of file
```

Comment:
There should be a newline at the end of file. It's a standard convention that every text file should end with a newline. This helps with version control systems and some tools to read and manipulate files.